From 4c7624b59b6a27abcc1616618c980e3c27660d77 Mon Sep 17 00:00:00 2001
From: William Bright <william.bright@imd-tec.com>
Date: Wed, 21 Oct 2026 15:18:29 +0100
Subject: [PATCH] metadata: add IMDT board and OEM

Register the IMDT SBC board (board sbc, id 0x20) and IMDT as an OEM
(oem-id 1) so UEFI FIT selection can match imdt,qcs6490-sbc-imdt and
imdt,qcs8550-sbc. The QCS8550 SoC node (msm-id 0x25b) this patch used to
add is upstream from v1.3.

Upstream-Status: Pending
Signed-off-by: William Bright <william.bright@imd-tec.com>
---
 qcom-metadata.dts | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/qcom-metadata.dts b/qcom-metadata.dts
index 5793487..5c14281 100644
--- a/qcom-metadata.dts
+++ b/qcom-metadata.dts
@@ -162,6 +162,10 @@
 		qrd {
 			board-id = <0x0000000b>;
 		};
+
+		sbc {
+			board-id = <0x00000020>;
+		};
 	};
 
 	boardrev {
@@ -294,5 +298,8 @@
 
 	oem {
 
+		imdt {
+			oem-id = <0x00000001>;
+		};
 	};
 };
